lib/access_import/access_import.rb

```diff
@@ -3,41 +3,71 @@ class AccessImport
 
   def initialize(path)
     @acdb = Mdb.open(path)
-    class_accessor(*[Laender, Sprachen, SprachKenntnisse, PersonenRolle, Plz, FamilienRollen,
-                     Begleitete, KostenTraeger, Journale, FallstelleTeilnehmer, Stundenerfassung, Ausbildungen,
-                     FreiwilligenEinsaetze, FreiwilligenEntschaedigung, Kontoangaben].map { |cl| cl.new(@acdb) })
+    class_accessor(
+      *[
+        Laender, Sprachen, SprachKenntnisse, PersonenRolle, Plz, FamilienRollen, Begleitete,
+        KostenTraeger, Journale, FallstelleTeilnehmer, Stundenerfassung, Ausbildungen,
+        FreiwilligenEinsaetze, FreiwilligenEntschaedigung, Kontoangaben
+      ].map { |cl| cl.new(@acdb) }
```

Comment:
I do need that array of instances. Its a parameter for `class_accessor(*accessors)`.
class_accors creates attr_reader and class variables from the classes name.